.spelling

```diff
@@ -612,6 +616,7 @@ Onboard
 onboarding
 Onboarding
 O'Neal
+onsite
```

Comment:
I normally use on-site. But it appears there have been some changes lately to use onsite if I search the web. It doesn't appear we have anything on istio.io pushing us one way or another, so this can be left alone.